pkg/cmd/util/clientcmd/factory.go

```diff
@@ -53,83 +57,115 @@ func DefaultClientConfig(flags *pflag.FlagSet) clientcmd.ClientConfig {
 // Factory provides common options for OpenShift commands
 type Factory struct {
 	*kubecmd.Factory
-	OpenShiftClientConfig clientcmd.ClientConfig
+	OpenShiftClientConfig ClientConfig
+}
+
+// intercept errors that need special treatment for end-users (e.g. no config file present)
```

Comment:
I don't see any reason to catch and replace error messages.  I think that's all this doing.